antlr-kotlin-runtime/src/commonMain/kotlin/com/strumenta/antlrkotlin/runtime/ext/Char.ext.kt

```diff
@@ -26,11 +29,7 @@ public fun Char.Companion.toChars(codePoint: Int, destination: CharArray, offset
 }
 
 public fun Char.Companion.charCount(codePoint: Int): Int =
-  if (codePoint >= MIN_SUPPLEMENTARY_CODE_POINT_) {
-    2
-  } else {
-    1
-  }
+  if (codePoint >= 0x10000 /* MIN_SUPPLEMENTARY_CODE_POINT */) 2 else 1
```

Comment:
I think removing braces is good, but the constant was making the code easier to read and maintain

Reply:
@ftomassetti Regarding constants, I agree that keeping them as `const val` makes the code more readable.  
My choice of manually inlining them comes from the fact an `internal const val` used outside of the file boundary produces additional accessors.

For example 
```kt
// A.kt
internal const val MIN_LOW_SURROGATE: Int = 0xDC00

// B.kt
if (x > MIN_LOW_SURROGATE) { }
```
translates to
```js
var MIN_LOW_SURROGATE = 56320;

function getMIN_LOW_SURROGATE() {
  return MIN_LOW_SURROGATE;
}

if (x > getMIN_LOW_SURROGATE()) { }
```
instead of
```js
if (x > 56320) { }
```

It is a subjective decision, so I'm up to reverting it, but it has its benefits.

Reply:
Sorry for the late reply. Yes, I would prefer you to revert it. I see your point, but I would favor readability over the slight performance hit

Reply:
@ftomassetti done. I've only removed the `inline` modifier, so that the compiler can at least partially inline constants.